src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -179,7 +190,8 @@ export function printTable( data: Object ) {
 export function processComponentOptionInput( passedParam: string, allowLocal: boolean ): ComponentConfig {
 	// cast to string
 	const param = passedParam + '';
-	if ( allowLocal && param.includes( '/' ) ) {
+	// This is a bit of a naive check
+	if ( allowLocal && param.includes( path.sep ) ) {
```

Comment:
What do you think about this:

```suggestion
	if ( allowLocal && /[\\\/]/.test(param) ) {
```

because on Windows, you can use both forward and back slashes :-) `C:/path/to/something` also works.

Reply:
That totally missed me, I'm gonna try and test it tomorrow